### PR TITLE
Closes #70: documentation update description and license files

### LINKED ISSUE #70: Documentation: Update DESCRIPTION and LICENCE files
### Please select a category the issue is focused on?

_No response_

### Let us know where something needs a refresh or put your idea here!

`DESCRIPTION`:
- [ ] Xiao Chen to be added as an author
- [ ] Order the authors list by last name - keeping the maintainer on top of the list
- [ ] Include Cytel Inc. as one of the license holders

`LICENSE`:
- [ ] Include Cytel Inc. as one of the license holders

`_pkgdown.yml`:
- [ ] Include Cytel Inc. as one of the license holders (+ import Cytel logo in `man/figures`)

### PR DESCRIPTION
**Pull request by Lina2689**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral family codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiral<ext> (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ recommend that a file or class name and description of purpose be included on
 the same "printed page" as the copyright notice for easier identification within
 third-party archives.
 
-    Copyright `Year: 2025` `License holders: F. Hoffmann-La Roche AG, Eli Lilly and Company`
+    Copyright `Year: 2025` `License holders: F. Hoffmann-La Roche AG, Eli Lilly and Company and Cytel Inc.`
```
Comment (Fanny-Gautier) [inline LICENSE.md:182]:
```suggestion
    Copyright `Year: 2025` `License holders: F. Hoffmann-La Roche AG, Eli Lilly and Company, and Cytel Inc.`
```

File: _pkgdown.yml [hunk 2]
```diff
@@ -17,7 +17,7 @@ news:
 authors:
   footer:
     roles: [aut]
-    text: "Developed by F. Hoffmann-La Roche AG, Eli Lilly and Company, "
+    text: "Developed by F. Hoffmann-La Roche AG, Eli Lilly and Company and Cytel Inc., "
```
Comment (Fanny-Gautier) [inline _pkgdown.yml:20]:
```suggestion
    text: "Developed by F. Hoffmann-La Roche AG, Eli Lilly and Company, and Cytel Inc., "
```

Comment (Fanny-Gautier) [reply _pkgdown.yml]: Without "and" before `Cytel Inc.` in `_pkgdown.yml`

File: _pkgdown.yml [hunk 3]
```diff
@@ -17,7 +17,7 @@ news:
 authors:
   footer:
     roles: [aut]
-    text: "Developed by F. Hoffmann-La Roche AG, Eli Lilly and Company, "
+    text: "Developed by F. Hoffmann-La Roche AG, Eli Lilly and Company, and Cytel Inc., "
```
Comment (Fanny-Gautier) [inline _pkgdown.yml:20]:
```suggestion
    text: "Developed by F. Hoffmann-La Roche AG, Eli Lilly and Company, Cytel Inc., "
```

File: DESCRIPTION [hunk 4]
```diff
@@ -4,16 +4,18 @@ Title: Neuroscience Extension Package for ADaM in 'R' Asset Library
...
+  person("Miles", "Almond", role = "aut", comment = c(ORCID = "0009-0007-1784-0355")),
+  person("Xiao", "Chen", role = "aut", comment = c(ORCID = "0009-0000-6959-5151")),
   person("Fanny", "Gautier", role = "aut", comment = c(ORCID = "0009-0004-3581-0131")),
   person("Gayatri", "G.", role = "aut"),
+  person("Meilin", "Jiang", role = "aut", comment = c(ORCID = "0000-0003-4515-4567")),
   person("Leena", "Khatri", role = "aut", comment = c(ORCID = "0000-0002-2268-4023")),
+  person("Edoardo", "Mancini", role = "aut", comment = c(ORCID = "0009-0006-4899-8641")),
+  person("Eric", "Nantz", role = "aut"),
+  person("Chris", "Pelentrides", role = "aut"),
   person("Lina", "Patil", role = "aut"),
```
Comment (Fanny-Gautier) [inline DESCRIPTION:16]: _Patil_ should come before _Pelentrides_

**Review by Fanny-Gautier — CHANGES_REQUESTED** [review]
Sorting order to update. Should be all good after this update. Thanks

**Review by manciniedoardo — APPROVED** [review]
